pulp_container/app/models.py

```diff
@@ -224,10 +224,10 @@ def create_namespace_group(self, permissions, parameters):
         group_type = parameters["group_type"]
         add_user_to_group = parameters["add_user_to_group"]
         group = Group.objects.create(
-            name="{}.{}.{}".format("container.namespace", group_type, self.name)
+            name="{}.{}.{}".format("container.namespace", group_type, self.pk)
```

Comment:
does anyone remember why we have decided to use namespace `name` and not `pk` for the namespace group creation? We use distribution pk for the distribution group creation.

Reply:
Because it was easier for people to recognize?

Reply:
I thought so too, just wanted to make sure there was not any other reason